This patch is similar to glibc-asflags.patch and is needed to change
the ld.so linker flags.  It is needed to work around this valgrind
bug: https://bugzilla.redhat.com/show_bug.cgi?id=1600034

diff --git a/Makeconfig b/Makeconfig
index 92e76d6200bbcd5b..376181c8ff06dea7 100644
--- a/Makeconfig
+++ b/Makeconfig
@@ -363,12 +363,12 @@ endif
 ifeq (yesyes,$(build-shared)$(have-z-combreloc))
 combreloc-LDFLAGS = -Wl,-z,combreloc
 LDFLAGS.so += $(combreloc-LDFLAGS)
-LDFLAGS-rtld += $(combreloc-LDFLAGS)
+override LDFLAGS-rtld += $(combreloc-LDFLAGS)
 endif
 
 relro-LDFLAGS = -Wl,-z,relro
 LDFLAGS.so += $(relro-LDFLAGS)
-LDFLAGS-rtld += $(relro-LDFLAGS)
+override LDFLAGS-rtld += $(relro-LDFLAGS)
 
 ifeq (yes,$(have-hash-style))
 # For the time being we unconditionally use 'both'.  At some time we
@@ -376,7 +376,7 @@ ifeq (yes,$(have-hash-style))
 # with --hash-style=gnu only.
 hashstyle-LDFLAGS = -Wl,--hash-style=both
 LDFLAGS.so += $(hashstyle-LDFLAGS)
-LDFLAGS-rtld += $(hashstyle-LDFLAGS)
+override LDFLAGS-rtld += $(hashstyle-LDFLAGS)
 endif
 
 ifeq (yes,$(enable-static-pie))
